spec/unit/pops/types/enumeration_spec.rb

```diff
@@ -1,50 +1,51 @@
 require 'spec_helper'
 require 'puppet/pops'
 
+module Puppet::Pops::Types
```

Comment:
Also, please note. I've been using this pattern for some time now. We currently have this pattern in at least six files on stable.